packages/manager/modules/pci/src/projects/project/workflow/add/schedule/schedule.html

```diff
@@ -0,0 +1,191 @@
+                    <span class="oui-icon oui-icon-error" aria-hidden="true"></span>
+                        <span data-translate="pci_workflow_create_cron_invalid"></span>
+                    </span>
+                </div>
+                <div class="oui-field__helper" data-translate="pci_workflow_create_cron_hour"></div>
+            </div>
+            <div class="oui-field pr-3" data-ng-class="{ 'oui-field_error': $ctrl.domInvalid }">
+                <label for="cron-dom"
+                    class="sr-only oui-field__label oui-label"
+                    data-translate="pci_workflow_create_cron_dom"></label>
+                <div class="oui-field__control oui-field__control_xs">
+                    <input
+                        class="oui-input"
+                        data-ng-class="{ 'oui-field_error': $ctrl.domInvalid }"
+                        type="text"
+                        id="cron-dom"
+                        name="cron-dom"
+                        required
+                        data-ng-model="$ctrl.schedule.cronPattern.dom"
+                        data-ng-blur="$ctrl.validateDom()" />
+                </div>
+                <div class="oui-field__error pr-2" data-ng-if="$ctrl.domInvalid">
+                    <span role="alert">
+                    <span class="oui-icon oui-icon-error" aria-hidden="true"></span>
+                        <span data-translate="pci_workflow_create_cron_invalid"></span>
+                    </span>
+                </div>
+                <div class="oui-field__helper" data-translate="pci_workflow_create_cron_dom"></div>
+            </div>
+            <div class="oui-field pr-3" data-ng-class="{ 'oui-field_error': $ctrl.monthInvalid }">
+                <label for="cron-month"
+                    class="sr-only oui-field__label oui-label"
+                    data-translate="pci_workflow_create_cron_month"></label>
+                <div class="oui-field__control oui-field__control_xs">
+                    <input
+                        class="oui-input"
+                        data-ng-class="{ 'oui-field_error': $ctrl.monthInvalid }"
+                        type="text"
+                        id="cron-month"
+                        name="cron-month"
+                        required
+                        data-ng-model="$ctrl.schedule.cronPattern.month"
+                        data-ng-blur="$ctrl.validateMonth()" />
+                </div>
+                <div class="oui-field__error pr-2" data-ng-if="$ctrl.monthInvalid">
+                    <span role="alert">
+                    <span class="oui-icon oui-icon-error" aria-hidden="true"></span>
+                        <span data-translate="pci_workflow_create_cron_invalid"></span>
+                    </span>
+                </div>
+                <div class="oui-field__helper">
+                    <span data-translate="pci_workflow_create_cron_month"></span>
+                    <oui-popover>
```

Comment:
It should be better to set `placement="bottom"` on cron popover to avoid to hide cron fields.